packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -89,6 +94,7 @@ const applyTransparentBackplateStyles: CSSRules<DesignSystem> = {
         [highContrastSelector]: {
             background: "ButtonFace !important",
             color: "ButtonText !important",
+            fill: "Highlight",

```

Comment:
Shouldn't the icon fill be the same color as the foreground text?

Reply:
This is a one off situation. in our Lightweight and Justified buttons, the beforeContent and afterContent do not get an underline indicator when a user hovers over it. It does slightly change color. I had to separate the color from the icon and the text, because on hover the text does not change, we just add an underline. If both the icon and text are the same color, and, they are both together, its fine when the user hovers, because you at least get an underline on the text. But if it is only the icon, and you hover over it, there is indication of hover., because the icon is is the same color in rest and hover state.
As I type this, I am thinking I can just change `color` to "Highlight", that should work.

Reply:
Sounds good. Maybe add a comment. That file is a little hard to parse because of those types of situations.

Reply:
updated